tests/data/conan_advisory.json

```diff
@@ -0,0 +1,2570 @@
+[
+    {
+        "test_index": 1,
+        "scheme": "conan1",
+        "native": ">=5.1.0 <=5.1.1",
```

Comment:
add a parameter ``` "gitlab_scheme" : "conan" ``` in every test here.

Reply:
@TG1999 Very interesting -- I'm getting a better sense of the structure we need for the gitlab advisory data.  I've followed your suggestion and the JSON objects now look like this:

```
  {
    "test_index": 1,
    "scheme": "conan1",
    "native": ">=5.1.0 <=5.1.1",
    "expected_vers": "vers:conan1/>=5.1.0|<=5.1.1",
    "gitlab_scheme": "conan"
  },
```